code/src/layouts/cart-drawer/cart-drawer-item/CartDrawerItem.scss

```diff
@@ -0,0 +1,49 @@
+@import "@design-system";
+
+.cart-item {
```

Comment:
Don't forget about spa prefix

Reply:
If I remember correctly, we use `spa` prefix only for components, prefixed with `App`?